lms/djangoapps/experiments/utils.py

```diff
@@ -76,5 +82,6 @@ def get_experiment_user_metadata_context(course, user):
         'course_start': course.start,
         'course_end': course.end,
         'has_staff_access': has_staff_access,
-        'forum_roles': forum_roles
+        'forum_roles': forum_roles,
+        'partition_groups': user_partitions,
```

Comment:
what does the string representation of partition groups look like?  
does it only contain a group id?  
I imagine if we could also get the group name that would be useful for debugging

Reply:
An example of the metadata now on my devstack: 


    {"username": "edx", "enrollment_mode": "audit", "upgrade_link": null, "user_id": 2, "course_start": "2016-01-01T00:00:00+00:00", "upgrade_deadline": null, "course_key_fields": {"org": "edX", "run": "course", "course": "E2E-101"}, "partition_groups": {"Enrollment Track Groups": [1, "Audit"]}, "course_end": "2018-12-31T00:00:00+00:00", "pacing_type": "instructor_paced", "upgrade_price": "Free", "course_id": "course-v1:edX+E2E-101+course", "has_staff_access": true, "forum_roles": [["Student"]], "enrollment_time": "2018-11-16T19:46:39.757856+00:00", "schedule_start": null, "email": "edx@example.com"}

Reply:
Note the partition group will tell you the name of the group, and then the user's group information in a list for that partition.